lib/autoupdate/agent/setup.go

```diff
@@ -85,6 +87,13 @@ WantedBy={{.TeleportService}}
 [Service]
 Environment="TELEPORT_UPDATE_CONFIG_FILE={{escape .UpdaterConfigFile}}"
 Environment="TELEPORT_UPDATE_INSTALL_DIR={{escape .InstallDir}}"
+`
+
+	deprecatedDropInTemplate = `# teleport-update
+# DO NOT EDIT THIS FILE
+[Service]
+ExecStart=
+ExecStart=-/bin/echo "The teleport-upgrade script has been disabled by teleport-update. Please remove the teleport-ent-updater package."
 `
```

Comment:
It might be better to make an override instead, so this nullifies the `WantedBy=` clause. We can also make an override on the timer itself to get rid of the `WantedBy=` as well.

Reply:
From what I can see, dependencies can only be added via drop-ins, not removed